src/wolff_client.erl

```diff
@@ -317,57 +310,54 @@ do_get_leader_connections(#{conns := Conns} = St, TopicOrAlias) ->
   maps:fold(F, [], FindInMap).
 
 %% return true if there is no need to refresh metadata because the last one is fresh enough
-is_metadata_fresh(#{metadata_ts := Topics, config := Config}, TopicOrAlias) ->
+is_metadata_fresh(#{metadata_ts := Topics, config := Config}, Topic) ->
```

Comment:
should not have named `TopicOrAlias` in the first place because this arg is always topic, but never aliased.

Reply:
It may be a `TopicOrAlias`: https://github.com/kafka4beam/wolff/blob/2bb47aa9b1f8578a07c3fcf7f79d3d8212b05c66/src/wolff_producers.erl#L101